azure-iot-device/samples/dps-cert-mgmt/dps_certificate_management.md

```diff
@@ -0,0 +1,106 @@
+# DPS CERTIFICATE MANAGEMENT
+
+In a nutshell, IoT Device can request a client certificate from a CA through DPS. 
+* Connect DPS to the private CA hosted by one of our CA partners. 
+* IoT device sends a CSR to DPS
+* DPS forwards it to the CA for signing and returns an X.509 client certificate to device. 
+* Device uses certificate to authenticate with IoT Hub.
+
+## Steps for making it work in preview (api-version=2021-11-01-preview)
+
+### Prerequisite Steps for DPS , enrollment and CA
+
+These steps must be done for any scenario inside DPS Cert Management.
+
+* A new DPS instance must created in __West Central US__.
+* Create an Enrollment Group or Individual Enrollment in this DPS instance.
+    * Choose to use symmetric key, TPM key or X.509 for this enrollment.
+    * Link an IoT Hub to the enrollment.
+* Create an internal-use test account with one of Microsoft CA partners by emailing _iotcerts@microsoft.com_
+    * Additionally, request them to add the DPS Service Endpoint to the allow list. `What allow list?`
+* Once the CA account is created there are 2 pieces of that will be provided.
+    * `api_key` -  DigiCert API key AND `profile_id` - DigiCert Client Cert Profile ID.
+* Use DPS Service API to associate the CA object above to the DPS. 
+    ```bash
+    curl -k -L -i -X PUT https://<dps_service_endpoint>/certificateAuthorities/<ca_name>?api-version=2021-11-01-preview -H 'Authorization: <service_api_sas_token>' -H 'Content-Type: application/json' -H 'Content-Encoding: utf-8' -d'{"certificateAuthorityType":"DigiCertCertificateAuthority","apiKey":"<api_key>","profileName":"<profile_id>"}'
+    ```
+   where,
+    * `dps_service_endpoint` - available in overview blade.
```

Comment:
>blade [](http://example.com/codeflow?start=53&length=5)

Does the reader know what a "blade" is and where to find it?

Reply:
Modified to say overview blade of dps details